src/services/tx/tx-sender/dispatch.ts

```diff
@@ -59,11 +59,11 @@ export const dispatchTxProposal = async ({
  */
 export const dispatchTxSigning = async (
   safeTx: SafeTransaction,
-  shouldEthSign: boolean,
+  isLegacySignMethod: boolean,
   txId?: string,
 ): Promise<SafeTransaction> => {
   const sdk = getAndValidateSafeSDK()
-  const signingMethod = shouldEthSign ? 'eth_sign' : 'eth_signTypedData'
+  const signingMethod = isLegacySignMethod ? ('eth_signTypedData_v3' as 'eth_signTypedData') : 'eth_signTypedData'
```

Comment:
This is internally using `"eth_sign"`. [The `safe-core-sdk`  uses it if the `signingMethod` isn't `"eth_signTypedData"`](https://github.com/safe-global/safe-core-sdk/blob/main/packages/safe-core-sdk/src/Safe.ts#L459-L464).

Reply:
I was just reviewing the forthcoming v1.0.0 support and [noticed that this will be extended](https://github.com/safe-global/safe-core-sdk/blob/safe-v1.0.0-compatibility/packages/safe-core-sdk/src/Safe.ts#L442-L446).